examples/interactivity/add-popups-onhover.html

```diff
@@ -85,7 +85,7 @@ <h1 class="as-title">Pop-ups on hover</h1>
         );
         await populationLayer.setPopupHover([
           'cartodb_id',
-          { attr: 'pop_2015', title: 'Population' }
+          { attr: 'pop_2015', title: 'Population', format: '~s' }
```

Comment:
Maybe we can add a comment to describe this is a d3.format. And a link to https://github.com/d3/d3-format for examples (as we will do in the docs in the future)

Reply:
Totally unrelated, but thinking of it, we could use title: null to completely hide the field, just displaying the 'attr' in the popup.